app/src/main/res/values/cnfg.xml

```diff
@@ -50,6 +50,7 @@ along with ProtonMail. If not, see https://www.gnu.org/licenses/.
         <item>íslenska</item>
         <item>Italiano</item>
         <item>日本語</item>
+        <item>Izwawen</item>
```

Comment:
Hello, The native name of kabyle is "Taqbaylit" note kabyle, please, would you fix that, than you.

Reply:
My apologies, I was not aware of that. I fixed it with an additional commit. Wish you a nice day! :)